src/Nordea/Components/FlatLink.elm

```diff
@@ -103,6 +109,47 @@ view attributes children (FlatLink config) =
         children
 
 
+inline : String -> String -> Html msg
+inline url text =
+    view [ href url ]
+        [ Html.text text ]
+        (FlatLink
+            { styles =
+                [ Css.display inlineFlex
+                , Css.fontWeight Css.inherit
+                , Themes.color Themes.PrimaryColor Colors.blueNordea
+                , hover [ Themes.color Themes.PrimaryColorLight Colors.blueDeep ]
+                , focus [ Themes.color Themes.PrimaryColorLight Colors.blueDeep ]
+                , visited [ Themes.color Themes.PrimaryColorLight Colors.purple ]
+                ]
+            , isDisabled = Just False
```

Comment:
With the interface change suggested above, this can be controlled by:
```suggestion
            , isDisabled = Just (List.member (disabled True) attrs)
```